ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListConnectorPluginsExecutor.java

```diff
@@ -43,18 +44,21 @@ public StatementExecutorResponse execute(
       final KsqlExecutionContext ksqlExecutionContext,
       final ServiceContext serviceContext
   ) {
-    final ConnectResponse<List<ConnectorPluginInfo>> plugins =
+    final ConnectResponse<List<PluginInfo>> plugins =
         serviceContext.getConnectClient().connectorPlugins();
     if (plugins.error().isPresent()) {
       return StatementExecutorResponse.handled(connectErrorHandler.handle(
           configuredStatement, plugins));
     }
 
     final List<SimpleConnectorPluginInfo> pluginInfos = new ArrayList<>();
-    for (final ConnectorPluginInfo info : plugins.datum().get()) {
+    for (final PluginInfo info : plugins.datum().get()) {
+      System.out.println("Info.type(): " + info.type() + " ConnectorType: "
```

Comment:
Is this meant to be here?

Reply:
Nope.  Was rushing yesterday to have something ready.:).